### ISSUE TITLE
Spoiler Support

### ISSUE THREAD

**Issue by practical-dreamer** [oobabooga/text-generation-webui#4822]
**Description**

I'd like to design an extension that adds the ability to hide output like spoilers. I'm experimenting with using this to allow the model a safe space to record an internal monologue which I do not want to see but I want the model to keep track of... A common way to do this using markdown language would be a simple `>! This is a Spoiler` followed by a hidden string of text. Platforms like discord also support this by wrapping your text in double pipes `|| this is a spoiler ||`. I've been trying to implement this using the `def output_modifier` method from the extension framework but it doesn't seem to be working.

**Additional Context**

Lots of use cases for this... you can use this to interact with layered characters that say one thing but think another... or could also use this to have a hidden space where the model thinks up unit test cases PRIOR to generating code to help it "think through" without me the user having to stare at the "rough draft"

**Bottom Line**

Can someone help me design an extension that hides all lines that begin with `>!` from the UI

**Comment by practical-dreamer** [comment]
bump

**Comment by practical-dreamer** [comment]
Got this to work kinda glitchy but used
```
blockquote {
    display: none;
}
```
added to css
